Sample Applications/WPFGallery/Views/AboutPage.xaml

```diff
@@ -48,7 +48,7 @@
         </StackPanel>
 
         <StackPanel Grid.Row="1" Margin="0,0,0,24">
-            <Expander>
+            <Expander AutomationProperties.Name="WPF Gallery Preview">
```

Comment:
@siagupta0202 - name property is not yet set. Test using accessibility insight tool if name property is getting set through your changes or not.

Please refer attached screenshot.

![Screenshot 2024-06-18 114834](https://github.com/microsoft/WPF-Samples/assets/85492722/bdd8a9c5-f371-4023-ad30-0f25e67e49ff)